surveys/2022-annual-survey/questions.md

```diff
@@ -572,21 +568,15 @@ Statements:
 - Rust makes me more productive
 - Rust is fun to use
 
-Rating:
-
-- Agree
-- Neither agree nor disagree
-- Disagree
-
 > **justification**
 >
 > There are several "truisms" about Rust that we'd like to get perspective on how true these are for users of Rust.
 >
 > Note that answers here can be subject to survivorship bias and so extra care should be taken with interpreting results.
 
-### In your opinion, how do you find the following aspects of Rust?
+### In your opinion, how would you prioritise work on the following aspects of Rust?
```

Comment:
I'm not sure I understand the question. Are expecting for the user to put these items in order? Does survey hero support this?